docker/supabase/supabase-production.yml

```
@@ -8,3 +8,7 @@ services:
     volumes:
       - ../../../supabase/volumes/db/data:/var/lib/postgresql/data
       - ./supabase/volumes/db/init:/docker-entrypoint-initdb.d
+
+  kong:
+    volumes:
+      - ../../../config/kong.yml:/var/lib/kong/kong.yml
```

```suggestion
      - ../../config/kong.yml:/var/lib/kong/kong.yml
```
and  `../../../supabase...` should be `../../supabase...`